index.js

```diff
@@ -138,10 +138,11 @@ export class UpdateAPK {
       console.log("RNUpdateAPK::getAppStoreVersion - iosAppId doesn't exist.");
       return;
     }
-    const URL =
-      "https://itunes.apple.com/us/app/apple-store/id" +
-      this.options.iosAppId +
-      "?mt=8";
+    //const URL =
```

Comment:
Don't leave commented out code, that's what version control is for - please remove the commented lines and just leave lines that are still doing work